components/SubmissionForm.vue

```diff
@@ -1,163 +1,185 @@
 <template>
-    <form class="flex flex-col place-self-center">
-        <h1
-            data-testid="submit-heading"
-            class="mb-3.5 text-center text-primary-text text-2xl font-bold font-sans leading-normal"
+    <div>
+        <form
+            v-show="isSubmissionFormReady"
```

Comment:
should we add a loader for this? (you could test using the chrome debugger tools fake 3g connection and see if the user is waiting on a blank page.

Reply:
@ermish That is a really cool tool I didn't know about. It is definitely necessary in that case